virttest/qemu_devices/qcontainer.py

```diff
@@ -2208,7 +2208,7 @@ def images_define_by_params(self, name, image_params, media=None,
     def serials_define_by_variables(self, serial_id, serial_type, chardev_id,
                                     bus_type=None, serial_name=None,
                                     bus=None, nr=None, reg=None,
-                                    bus_extra_params=''):
+                                    bus_extra_params='', bus_add=None):
```

Comment:
The new parameter indicate if reuse the existing `virtio-serial-pci` bus for `virtserialport` type serial.
How about `share_bus` or `reuse_bus`  ?
And it is a boolean, better use `True` instead of `None`. Means reuse the existing serial bus by default.